mtags/src/main/scala/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -26,6 +26,12 @@ class CompletionProvider(
       case '{' if params.text().charAt(i - 1) == '$' =>
         // Insert potentially missing `}` to avoid "unclosed literal" error in String interpolator..
         CURSOR + "}"
+      case '*'
+          if params
+            .text()
+            .charAt(i - 1) == '*' && params.text().charAt(i - 2) == '/' =>
+        // Insert potentially missing `*/` to avoid comment out all codes after the "/**".
+        CURSOR + "*/"
```

Comment:
If we don't close the comment here, the whole code below the `/**` will be commented out, and we cannot traverse the AST (retrieved from a compilation unit) below the comment